packages/flutter/lib/src/widgets/text.dart

```diff
@@ -647,14 +650,16 @@ class Text extends StatelessWidget {
       effectiveTextStyle = effectiveTextStyle!.merge(const TextStyle(fontWeight: FontWeight.bold));
     }
     final SelectionRegistrar? registrar = SelectionContainer.maybeOf(context);
+    final GlobalKey? textKey = registrar == null ? null : GlobalKey();
+    final _SelectableTextContainerDelegate? selectionDelegate = registrar == null ? null : _SelectableTextContainerDelegate(textKey!);
```

Comment:
When does it happen that there is no SelectionContainer above this, but we still need selectionDelegate?

Reply:
This would never happen, if I am understanding correctly. The delegate will be null when there is no `SelectionContainer` above this.